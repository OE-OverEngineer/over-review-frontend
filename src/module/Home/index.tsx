import React from 'react';

import { Layout } from 'antd';
import { Content, Footer } from 'antd/lib/layout/layout';

import Star from 'common/assets/images/star.svg';
import Navbar from 'common/components/Navbar';

import BannerSlider from './components/BannerSlider';
import DiscoverMovie from './components/DiscoverMovie';

const Home: React.FC = () => {
  return (
    <Layout>
      <Star className="absolute h-full w-full z-0" />
      <Navbar />
      <Content>
        <BannerSlider />
        <DiscoverMovie />
      </Content>
      <Footer className="text-center">Over Review Â©2021 Created by Over Engineer</Footer>
    </Layout>
  );
};

export default Home;
