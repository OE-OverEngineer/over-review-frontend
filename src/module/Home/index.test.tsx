import React from 'react';

import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import Home from '.';

describe('Index page', () => {
  it('Index page should be defined', () => {
    const wrap = shallow(<Home />);
    // console.log(wrap);

    expect(wrap.exists()).toBe(true);
    expect(toJson(wrap)).toMatchSnapshot();
  });
});
