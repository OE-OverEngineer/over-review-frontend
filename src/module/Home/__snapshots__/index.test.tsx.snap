// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index page Index page should be defined 1`] = `
<div>
  <Head>
    <title>
      Home Page
    </title>
    <link
      href="/over_icon.svg"
      rel="icon"
    />
  </Head>
  <Layouts
    router={
      Array [
        "home",
      ]
    }
  >
    <BannerSliderSection />
    <DiscoverMovieSection />
    <HallOfFameSection />
  </Layouts>
</div>
`;
